swatch-contracts/src/main/resources/META-INF/openapi.yaml

```diff
@@ -187,17 +187,37 @@ paths:
         - support: [ ]
         - test: [ ]
       operationId: syncSubscriptionsForContractsByOrg
+  /api/swatch-contracts/internal/rpc/reset/{org_id}:
+    description: "Clear all contracts and subscriptions for a given org ID. "
+    delete:
+      parameters:
+        - name: org_id
+          description: "Organization ID"
+          schema:
+            type: string
+          in: path
+          required: true
+      summary: "Clear all contracts and subscriptions for given org_id."
+      responses:
+        '204':
+          description: "Contracts and subscriptions successfully deleted"
+      security:
+        - support: [ ]
+        - test: [ ]
+        - service: [ ]
+      operationId: deleteContractsByOrg
```

Comment:
The method name implies that we're going to delete only the contracts. 
I would suggest renaming to `deleteDataForOrg`, doing this would also avoid the collision with the existing deleteContractsByOrg